### DISCUSSION TITLE
add shortcut

### DISCUSSION THREAD

**Discussion by faon-92** [SubtitleEdit/subtitleedit#7994] [Q&A]
Hi,

It happens often to me that the whisper transcribe the correct text but put them to a wrong line. More precisely, it put several phrases into a same line...
Is it possible to create a shortcut to "move text after cursor position to next subtitle and offset the rest"?

thanks for your reply,

**Comment by darnn** [comment]
Are you talking about `Split line at cursor position` (under `Text box` in shortcuts)? Or are you saying that you want to only keep the text before the cursor in the subtitle, and move the rest of the text to the next subtitle, which already has text in it, but not change the timing? If it's the latter, isn't that what `Move text after cursor position to next subtitle and go to next` does?

**Comment by faon-92** [comment]
not really. 
I give an example to illustrate:
subtitle 1: Hello
subtitle 2: I'm faon-92
subtitle 3: nice to meet you
subtitle 4:...

It happens often that whisper gives:
subtitle 1: Hello I'm faon-92
subtitle 2: nice to meet you
subtitle 3:...

when I use "Move text after cursor position to next subtitle and go to next" in the line 1
it gives out:
subtitle 1: Hello
subtitle 2: I'm faon-92 nice to meet you
subtitle 3:...

when there are many subtitles, I need to adjust manually until the last subtitle.

Is it possible to have "move text after cursor position to next subtitle and offset the rest" which gives out:
Subtitle 1: hello
subtitle 2: I'm faon-92
subtitle 3: nice to meet you
subtitle 4: ...
all that without changing the Timing, set by me manually and respected when Whipser transcribe the selected line from audio to text?

Many thanks,

**Comment by darnn** [comment]
Ah, well, in that case, all I can add is that you can use `Column, insert text` to move all the text down one subtitle without changing the existing timecodes.

**Comment by faon-92** [comment]
ok... I figured it out now, it's a huge help for me, Thank you so much @darnn
